main.go

```diff
@@ -173,52 +214,97 @@ func getProjectID(baseURL string, token string, projectname string, namespace st
 }
 
 // Get and return currently active milestones
-func getActiveMilestones(baseURL string, token string, projectID string) ([]milestoneAPI, error) {
-	state := "active"
-	return getMilestones(baseURL, token, projectID, state)
+func getActiveMilestones(baseURL string, token string, projectID string, api string) ([]milestoneAPI, error) {
```

Comment:
Move from projectID to project to keep a similar function input as getInactiveMilestones.